gulpfile.babel.js

```diff
@@ -235,12 +234,18 @@ gulp.task( 'jshint', gulp.parallel( 'js:hint' ) );
 // Default task
 gulp.task(
 	'default',
-	gulp.parallel( react_build, sass_build, 'old-styles', 'js:hint', 'php:module-headings', 'gutenberg:blocks' )
-);
-gulp.task(
-	'watch',
-	gulp.parallel( react_watch, sass_watch, 'old-styles:watch' )
+	gulp.series(
```

Comment:
For the reviewers: Prettier made quite a few changes in the file format, but this is the only part that really changed.